extensions/2.0/Vendor/EXT_lights_image_based/README.md

```diff
@@ -0,0 +1,136 @@
+1. JPEG's - assumed to be regular, LDR, RGB images
+2. PNG's - assumed to be HDR images packed in RGBD format.
+
+### RGBD
+RGBD is a way to pack HDR data so that the alpha channel contains a value that, when the RGB channels are divided by it, can produce values greater than 1. Runtimes can choose to unpack this data to float textures at load time (recommended) or keep it packed on the GPU (and unpack in the shader) to save memory.
+
+RGBD has several advantages over similar packing schemes like RGBM and RGBE.
+
+1. Simple to pack and unpack. Cheaper than RGBE.
+2. Provides better range than RGBM.
+3. Interpolates better than RGBE. Important if the data is kept packed during rendering and filtering is used. Note that the interpolation isn't perfect so visual artifacts may still appear. This is why unpacking to float at load-time is recommended.
+4. Packed PNG doesn't appear as garbled as a one packed with RGBE.
+
+If the referenced image is a 4-channel PNG (i.e. with transparency), it will be assumed to contain RGBD data. If the image is only 3 channels, it will be assumed to contain LDR lighting data.
+
+### Future HDR Image Support
+This extension expects 4-channel PNG's to contain RGBD data. However, this should not interfere in any future glTF extensions that provide HDR support. e.g. if there is an extension that defines RGBE packing for PNG's that is in use for the texture/image, that should override the assumed RGBD packing. Support for other HDR file types (e.g. CTTF formats or .hdr/.exr files) should not interfere either as they won't be PNG's.
+
+## Adding Light Instances to Scenes
+
+Each scene can have a single IBL light attached to it by defining the `extensions.EXT_lights_image_based` property and, within that, an index into the `lights` array using the `light` property.
+
+```javascript
+"scenes" : [
+    {
+        "extensions" : {
+            "EXT_lights_image_based" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+### Image-Based Light Properties
+
+| Property | Description | Required |
+|:-----------------------|:------------------------------------------| :--------------------------|
+| `name` | Name of the light. | No, Default: `""` |
+| `rotation` | Quaternion that represents the rotation of the IBL environment. | No, Default: `[0.0, 0.0, 0.0, 1.0]` |
+| `intensity` | Brightness multiplier for environment. | No, Default: `1.0` |
+| `irradianceCoefficients` | Declares spherical harmonic coefficients for irradiance up to l=2. This is a 9x3 array. | :white_check_mark: Yes |
+| `specularImages` | Declares an array of the first N mips of the prefiltered cubemap. Each mip is, in turn, defined with an array of 6 images, one for each cube face. i.e. this is an Nx6 array. | :white_check_mark: Yes |
+| `specularImageSize` | The maximum dimension (in pixels) of the first specular mip. This is needed to determine, pre-load, the total number of mips needed. | :white_check_mark: Yes |
```

Comment:
Assuming that images must be squares (btw, the spec should explicitly say that), what does "maximum dimension" mean for the first mip? Also, are npot images supported?

Reply:
I added the stipulation that they must be square and power-of-two.